Source/objects.cpp

```diff
@@ -2603,23 +2612,21 @@ void OperateL1LDoor(int pnum, int oi, bool sendflag)
 
 void OperateL2RDoor(int pnum, int oi, bool sendflag)
 {
-	int xp, yp;
-	bool dok;
+	const Point &objPos = object[oi].position;

```

Comment:
We are trying to avoid using abbreviations, so either `objectPosition` or `objPosition` would be preferred.

Reply:
Done :). as `objectPosition` isn't much shorter than `object[oi].position` I used a reference to the current object and called it `door` instead to hopefully make it more clear what we're operating on. This also let me make these functions consistently use the reference instead of mixing indexes into the objects array and the position property reference.